io.github.mrmyhuang.cbetar2.yml

```diff
@@ -0,0 +1,74 @@
+app-id: io.github.mrmyhuang.cbetar2
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=host
```

Comment:
this is a very broad permission.

Reply:
- --socket=pulseaudio: the app uses text-to-speech function.
  - --share=network: this app needs Internet to access the Buddhism text database.
  - --device=dri: deleted.
  - --filesystem=host: change host to home

Reply:
actualy for electron DRI is likely required. If you get messages about about GL or libdrm, it's because of that.

Reply:
I try to build and run my app with and without dri, but the terminal messages (including errors) seems to be no different... However, my app works well in both cases. Thus, I remove the permission. Thank you.